sdk/test-utils/perfstress/GettingStarted.md

```diff
@@ -0,0 +1,258 @@
+
+2. It is expected that the track-1 perf tests are counterparts of track-2 tests, so they need to have the same names as specified in the track-2 tests for convenience.
+
+3. Add a `package.json` such as [example-track-1-perf-package.json](https://github.com/Azure/azure-sdk-for-js/blob/fe9b1e5a50946f53b6491d7f67b2420d8ee1b229/sdk/storage/perf-tests/storage-blob-track-1/package.json) at `sdk/<service>/perf-tests/<service-sdk>` folder.
+
+   Make sure to import your `<service-sdk>` and the `test-utils-perfstress` project.
+
+   ```json
+     "devDependencies": {
+        "@azure/<service-sdk>": "^<latest-track-1-version>",
+        "@azure/test-utils-perfstress": "file:../../../test-utils/perfstress/azure-test-utils-perfstress-1.0.0.tgz",
+      }
+   ```
+
+   Set the name of the package and mark it as private.
+
+   ```json
+    "name": "@azure-tests/perf-<service-sdk>-track-1",
+    "private": true,
+   ```
+
+   _Note: Track-1 packages will not be managed by `rush`, instead `npm` will be used to manage/run the track-1 tests, you can copy the readme such as the [storage-blob-perf-tests-track-1-readme](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/storage/perf-tests/storage-blob-track-1/README.md) for instructions._
+
+   Make sure to add the "setup" step in package.json.
+
+   ```json
+       "setup": "node ../../../../common/tools/perf-tests-track-1-setup.js",
+   ```
+
+4. Run `rush update` followed by `npm run setup` to be able to use the perf framework for track-1 perf tests.
+
+   _`npm run setup` installs the dependencies specified in `package.json`_
+
+5. Repeat the step 6 from the previous section for the track-1 too to get the `tsconfig.json`, `sample.env`(and `.env`) files.
+
+## [Writing perf tests](#writing-perf-tests)
+
+### [Entry Point](#Entry-point)
+
+Add an `index.spec.ts` at `sdk/<service>/perf-tests/<service-sdk>/test/`.
+
+```js
+import { PerfStressProgram, selectPerfStressTest } from "@azure/test-utils-perfstress";
+import { `ServiceNameAPIName`Test } from "./api-name.spec";
+import { `ServiceNameAPIName2`Test } from "./api-name2.spec";
+
+console.log("=== Starting the perfStress test ===");
+
+const perfStressProgram = new PerfStressProgram(selectPerfStressTest([`ServiceNameAPIName`Test, `ServiceNameAPIName2`Test]));
+
+perfStressProgram.run();
+```
+
+### [Base Class](#Base-Class)
+
+Base class would have all the common code that would be repeated for each of the tests - common code such as creating the client, creating a base resource, etc.
+
+Create a new file such as `serviceName.spec.ts` at `sdk/<service>/perf-tests/<service-sdk>/test/`.
+
+```js
+import { PerfStressTest, getEnvVar } from "@azure/test-utils-perfstress";
+import {
+  ServiceNameClient
+} from "@azure/<service-sdk>";
+
+// Expects the .env file at the same level
+import * as dotenv from "dotenv";
+dotenv.config();
```

Comment:
It feels the entry point is a more appropriate home for this because nothing about this is abstract.

Reply:
Can move, thanks!